packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -394,28 +394,6 @@ export class TokenDetectionController extends StaticIntervalPollingController<To
         }
       },
     );
-
-    this.messagingSystem.subscribe(
```

Comment:
Explanation: we don't need to handle this case since the detection is multichain now , this is why we're removing this code